src/suite.js

```diff
@@ -14,26 +14,33 @@ const defaultOptions = {
   minSamples: 20
 };
 
+const targetList = [
+  "acorn",
+  "babel",
+  "babylon",
+  "buble",
+  "chai",
+  "coffeescript",
+  "espree",
+  "esprima",
+  "jshint",
+  "lebab",
+  "prepack",
+  "prettier",
+  "source-map",
+  "typescript",
+  "uglify-es",
+  "uglify-js"
+];
```

Comment:
Extract this list into its own file, and make it export a `Set` instead of an array.